source/common/router/header_formatter.cc

```diff
@@ -28,23 +29,27 @@ std::string formatUpstreamMetadataParseException(const std::string& params,
   return fmt::format("Incorrect header configuration. Expected format "
                      "UPSTREAM_METADATA([\"namespace\", \"k\", ...]), actual format "
                      "UPSTREAM_METADATA{}{}",
-                     params, reason);
+                     absl::StrCat(params), reason);
```

Comment:
You are just trying to convert a string_view to a std::string?  Use std::string(params)